src/vs/editor/common/core/characterClassifier.ts

```diff
@@ -69,11 +72,17 @@ export class CharacterSet {
 		this._actual = new CharacterClassifier<Boolean>(Boolean.False);
 	}
 
-	public add(charCode: number): void {
-		this._actual.set(charCode, Boolean.True);
+	public add(char: string): void {
+		if (char.length > 0) {
+			this._actual.set(char.charCodeAt(0), Boolean.True);
```

Comment:
Unsure what happens with characters that are longer than `1`, like emojis etc. This won't change todays behaviour because all call use `charCodeAt(0)` but might be conceptually wrong in the util here